driver/src/main/java/org/neo4j/driver/internal/handlers/pulln/BasicPullResponseHandler.java

```diff
@@ -267,13 +229,217 @@ private void dispose()
         this.summaryConsumer = null;
     }
 
-    protected Status status()
+    protected State getState()
```

Comment:
The driver uses naming style `void status(Status s)`, and `Status status()`. This is because when we are tying to give a unified API across all languages, `getXX` and `setXX` does not apply in many languages such as python or Js. So we decided we use getter and setters in this driver without `get` and `set`. I would suggest to change the name back so that we keep the naming style in the whole codebase to be consistent.